guides/common/modules/con_additional-maintenance-required-for-load-balancing-in-project.adoc

```diff
@@ -0,0 +1,10 @@
+[id="additional-maintenance-required-for-load-balancing-in-project_{context}"]
+= Additional maintenance required for load balancing in {Project}
```

Comment:
Just a general comment.

Whenever I see "in {Project}" or "for {Project}", I wonder whether it is necessary. Our module titles are very long, which makes them harder to scan. I think the context makes "in/for {Project}" unnecessary in most cases.

I am thinking about opening an issue to make our headers more concise. :-)

Reply:
@apinnick It's good that you're bringing this up. I lean towards more specific headings that include the product name to ensure reusable and context-free modules[1]. I know we don't have tooling to support proper reuse and searchability (yet!), but to me, that's a chicken-and-egg problem: Requests for that tooling are sometimes downplayed with the argument that module reuse is not that common, and trying to write content for reuse hits pushback with the argument that our tooling can't leverage it anyway :shrug: 

+1 on opening an issue to discuss this and reach some sort of agreement :)

[1] "Moreover, because modules are written as context-free elements independent of other modules, they are re-usable." as specified in the [mod docs guide](https://redhat-documentation.github.io/modular-docs/#modular-docs-terms-definitions).

Reply:
I understand the logic behind including the product name in the header. At the same time, there does not appear to be a lot of module reuse. Wasn't that the reason we decided to remove the contexts? :-)

Reply:
I can see both sides to this. However, I feel like this is out of the scope of this PR unless headers have been changed systematically. Can you please open an issue to track and discuss this @apinnick ?

Reply:
Done . https://github.com/theforeman/foreman-documentation/issues/2956